ionoscloud/resource_ipfailover.go

```diff
@@ -28,6 +34,7 @@ func resourceLanIPFailover() *schema.Resource {
 			"ip": {
 				Type:             schema.TypeString,
 				Required:         true,
+				ForceNew:         true,
 				ValidateDiagFunc: validation.ToDiagFunc(validation.All(validation.IsIPAddress)),
 			},
 			"nicuuid": {
```

Comment:
a description would be good

Reply:
@cristiGuranIonos done in 955c632a395828c57c898b64b403658977067c9f